grouper/fe/handlers.py

```diff
@@ -96,8 +98,9 @@ def get(self, user_id=None, name=None):
 
         try:
             user_md = self.graph.get_user_details(user.name)
-        except KeyError:
-            # User is probably very new, so they have no metadata yet.
+        except (KeyError, NetworkXError):
```

Comment:
I'm not 100% sure where the KeyErrors come from but I'd guess from the comment that it's graph.py line 507 in the get_user_details function.  Likewise, NetworkXError is coming from line 475.  Maybe we should just make that function do the right thing instead.